src/modules/fancyzones/FancyZonesLib/CompositionDrawing.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include "Drawing.h"
+
+#include <winrt/base.h>
+#include <Windows.h>
+#include <dxgi1_3.h>
+#include <d3d11_2.h>
+#include <d2d1_3.h>
+#include <d2d1_3helper.h>
+#include <d2d1helper.h>
+#include <dcomp.h>
+#include <dwmapi.h>
+#include <string>
+
+class CompositionDrawing : public Drawing
+{
+public:
+    void Init(HWND window);
```

Comment:
It doesn't make sense to use this class if it wasn't inited. So let's the `Init` returns bool at least. The best case IMO would be the RAII principle.

Reply:
To check if the class is inited one should use the bool operator. Returning a bool here would mean the `TabsZoneTitleBar` and others would have to hold both `m_drawing` and the bool. The thing is I should call `Init` on `WM_CREATE`, and not from the constructor.